api/tests/test_assets_assignee_api.py

```diff
@@ -0,0 +1,126 @@
+from unittest.mock import patch
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+
+from core.models import (
+    User, AssetAssignee, Department, OfficeBlock,
+    OfficeFloor, OfficeFloorSection, AssetCategory,
+    AssetSubCategory, AssetType, AssetMake, AssetModelNumber, Asset,
+    AllocationHistory,
+)
+
+from api.tests import APIBaseTestCase
+client = APIClient()
+
+
+class AssetAssigneeAPITest(APIBaseTestCase):
+    """ Tests for the AssetAssignee endpoint"""
+
+    def setUp(self):
+        super(AssetAssigneeAPITest, self).setUp()
+        self.user = User.objects.create(
+            email='testuser@gmail.com', cohort=19,
+            slack_handle='tester', password='qwerty123'
+        )
+
+        self.department = Department.objects.create(name="Finance")
+        self.office_block = OfficeBlock.objects.create(name='Andela Tower')
+        self.office_floor = OfficeFloor.objects.create(
+            block=self.office_block,
+            number=14
+        )
+
+        self.office_section = OfficeFloorSection.objects.create(
+            name='Safari',
+            floor=self.office_floor
+        )
+
+        self.asset_category = AssetCategory.objects.create(
+            category_name="Accessories")
+
+        self.asset_sub_category = AssetSubCategory.objects.create(
+            sub_category_name="Sub Category name",
+            asset_category=self.asset_category)
+        self.asset_type = AssetType.objects.create(
+            asset_type="Asset Type",
+            asset_sub_category=self.asset_sub_category)
+        self.make_label = AssetMake.objects.create(
+            make_label="Asset Make", asset_type=self.asset_type)
+        self.assetmodel = AssetModelNumber.objects.create(
+            model_number="IMN50987", make_label=self.make_label)
+        self.assetmodel_2 = AssetModelNumber.objects.create(
+            model_number="IMN509889", make_label=self.make_label)
+
+        self.asset = Asset.objects.create(
+            asset_code="IC001",
+            serial_number="SN001",
+            purchase_date="2018-07-10",
+            current_status="Available",
+            model_number=self.assetmodel
+        )
+
+        self.asset_2 = Asset.objects.create(
+            asset_code="IC002",
+            serial_number="SN002",
+            purchase_date="2018-07-12",
+            current_status="Available",
+            model_number=self.assetmodel_2
+        )
+
+        self.allocation_user = AllocationHistory.objects.create(
+            asset=self.asset,
+            current_owner=self.user.assetassignee
+        )
+
+        self.allocation_department = AllocationHistory.objects.create(
+            asset=self.asset_2,
+            current_owner=self.department.assetassignee
+        )
+
```

Comment:
Allocate an asset to a workspace to increase test coverage on `get_assignee()` on **AssetAssigneeSerializer**.

Reply:
One more thing @QUDUSKUNLE @precious-ijege ☝️